ITE/10/README.adoc

```diff
@@ -194,32 +205,30 @@ func Verify(layoutEnvelope, layoutKeys, attestations, now) {
     }
 
     for _, inspection := range layout.Inspections {
-        expectedPredicateTypes := []string{}
-        for _, expectedPredicate := range inspection.ExpectedPredicates {
-            expectedPredicateTypes = append(expectedPredicateTypes, expectedPredicate.PredicateType)
-        }
-        inspectionAttestations := executeInspection(inspection.Command, expectedPredicateTypes) // {predicateType: attestation}
-        allAttestations := combineAttestations(inspectionAttestations, attestations)
-        for _, expectedPredicate := range inspection.ExpectedPredicates {
-            attestation := inspectionAttestations[expectedPredicate.PredicateType]
-            applyMaterialRules(inspection.ExpectedMaterials, attestation.Materials, allAttestations)
-            applyProductRules(expectedPredicate.ExpectedProducts, attestation.Products, allAttestations)
-        }
+        inspectionAttestation := executeInspection(inspection.Command)
+        applyMaterialRules(inspection.ExpectedMaterials, inspectionAttestation.Materials, attestations)
+        applyProductRules(inspection.ExpectedProducts, inspectionAttestation.Products, attestations)
     }
 }
 ```
 
-[[motivation]]
-== Motivation
+=== Custom Verifiers
+
+As noted above, in-toto v1.0's verification workflow generates an in-toto
+summary link if verification succeeds. This is a powerful semantic that allows
+features like sublayouts that allow for delegating the supply chain definition
+for a step to authorized functionaries, as well as summarized verification for
+environments with limited resources. With the introduction of in-toto
```

Comment:
Nit: clunky sentence "...semantic that allows features like sublayouts that allow for..."